### ISSUE TITLE
Error enabling backups: Unable to load resource world\dot-gitignore

### ISSUE THREAD

**Issue by Notexe** [pcal43/fastback#158]
Hello,
The following error triggers after running the enable command:
```
[00:16:36] [pool-2-thread-1/ERROR]: Error enabling backups
java.io.FileNotFoundException: Unable to load resource world\dot-gitignore
	at net.pcal.fastback.utils.FileUtils.writeResourceToFile(FileUtils.java:53) ~[fastback-0.7.0+1.19.2.jar:?]
	at net.pcal.fastback.WorldConfig.doWorldMaintenance(WorldConfig.java:192) ~[fastback-0.7.0+1.19.2.jar:?]
	at net.pcal.fastback.commands.EnableCommand.lambda$enable$1(EnableCommand.java:62) ~[fastback-0.7.0+1.19.2.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
[00:16:36] [Render thread/INFO]: [CHAT] An unexpected backup error occurred. See log for details.
```
I've checked the jar file for the mod and that file does exist under the world folder.

**Comment by pcal43** [comment]
Thanks.  You're on Windows, I take it?

Pretty sure this is a regression that went in version `0.6.0`.  I'll get a fix out shortly.  Version `0.5.1` should also work for you in the meantime.

**Comment by Notexe** [comment]
Yeah I am using Windows. I forgot to mention that originally. Thank you, I'll use 0.5.1 for now.

**Comment by pcal43** [comment]
should be fixed in 0.7.1

**Comment by Notexe** [comment]
Thanks, it works now 😄
